src/network/requests.ts

```diff
@@ -46,3 +46,12 @@ export const deleteMovie = (
     .then(response => response.json())
     .then(() => fetchMovies(user, setMovies));
 };
+
+export const getRecommendations = (emails: string) => {
+  const searchParams = new URLSearchParams({ emails });
+  return fetch("/get-recommendation?" + searchParams)
```

Comment:
We can use async/await here if we want!

```js
const response = fetch("/get-recommendation?" + searchParams);
return response.json().movies;
```

Also probably want to add some error handling here, like `if (response.status !== 200) { alert('Failed to get recommendations') }` or something like that